bluesnap/nodes.py

```diff
@@ -54,8 +54,10 @@ def _load_population(h5_filepath, csv_filepath, population):
     result = pd.DataFrame(index=np.arange(node_count))
 
     _all = libsonata.Selection([(0, node_count)])
-    for prop in sorted(nodes.attribute_names):
-        result[prop] = nodes.get_attribute(prop, _all)
+    for attr in sorted(nodes.attribute_names):
+        result[attr] = nodes.get_attribute(attr, _all)
+    for attr in sorted(nodes.dynamics_attribute_names):
```

Comment:
Alternatively, we can let this SONATA idiosyncrasy leak and expose attributes from `dynamics_parameter` in a separate `NodePopulation` method.

Reply:
I am not sure about this. As a user I don t think I would like to learn which attributes are dynamic or not.